lib/chef/provisioning/convergence_strategy/install_msi.rb

```diff
@@ -9,8 +10,6 @@ def initialize(convergence_options, config)
         super
         @chef_version ||= convergence_options[:chef_version]
         @prerelease ||= convergence_options[:prerelease]
-        @install_msi_url = convergence_options[:install_msi_url] || 'https://www.chef.io/chef/install.msi'
```

Comment:
Yeah, we can support the option still, but the behaviour will change a little - rather than always installing the latest version, windows will now install the version the user tells us to if they don't give a specific msi install url. 
If they do, we can just grab that file as before.